packages/amplify_datastore/lib/method_channel_datastore.dart

```diff
@@ -231,6 +252,23 @@ class AmplifyDataStoreMethodChannel extends AmplifyDataStore {
     return serializedItem["modelName"] as String;
   }
 
+  AmplifyException _deserializeExceptionFromMap(Map<String, dynamic> e) {
+    if (e['errorCode'] == 'DataStoreException') {
+      return DataStoreException.fromMap(Map<String, String>.from(e['details']));
+    } else if (e['errorCode'] == 'AmplifyAlreadyConfiguredException') {
+      return AmplifyAlreadyConfiguredException.fromMap(
+          Map<String, String>.from(e['details']));
+    } else {
```

Comment:
It looks like the `else` covers use case that `e['errorCode'].isEmpty !== true`, with which there is an error code, but not handled by this `_deserializeExceptionFromMap` method, for this case categorizing them as "unhandled" could be more accurate than "missingExceptionMessage"?

Reply:
Hmm makes sense.  However when looking at: 

```
  static const missingExceptionMessage =
      'An unknown exception has happened. ' + _createGithubIssueString;

```

It appears the existing error message returned is sufficient?  Do you think we should be saying "An unhandled exception has happened" instead? 

@HuiSF